src/scripts/main.js

```diff
@@ -1,5 +1,26 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
```

Comment:
The '.spider' and '.wall' elements are being selected every time the script runs. It would be more efficient to select these elements once and store them in variables, then reference those variables in your code.

Reply:
> Елементи '.spider' і '.wall' вибираються під час кожного запуску сценарію. Було б ефективніше вибрати ці елементи один раз і зберегти їх у змінних, а потім посилатися на ці змінні у вашому коді.

I've done the same, I don't understand you